### PR TITLE
feat(exports tables): exports table implemented

### PR DESCRIPTION
**Pull request by alebinson**
This PR relates to issue #26 
Final look and fill is:
image

Please pay attention on Status and Date column:
1. **Status** - textual, is it enough?
2. **Date** what is the date format DD/MM/YYYY HH:mm ?

### CODE REVIEW

**Comment by ssharon-rnd** [comment]
Look great !!!
let's keep like this

________________________________

**Comment by alebinson** [comment]
@ssharon-rnd  U R fast!

File: src/common/i18n/he.json [hunk 1]
```diff
@@ -32,6 +33,16 @@
     "export.dialog-info.link.label": "קישור:",
     "export.dialog-info.link.pattern": "https://packages/{packageName}.gpkg",
 
+    "export-table.dialog.title": "חבילות",
+    "export-table.link.text": "הורד",
+    "export-table.table-column-header.fileName.text": "שם הקובץ",
+    "export-table.table-column-header.sizeEst.text": "(Mb)גודל משואר",
```
Comment (asafMasa) [inline src/common/i18n/he.json:39]: spelling

File: src/common/i18n/he.json [hunk 2]
```diff
@@ -32,6 +33,16 @@
...
+    "export-table.table-column-header.tilesEst.text": "מס' טיילים משואר",
```
Comment (asafMasa) [inline src/common/i18n/he.json:40]: spelling

File: src/common/i18n/he.json [hunk 3]
```diff
@@ -32,6 +33,16 @@
...
+    "export-table.table-column-header.status.text": "סטטוס",
```
Comment (asafMasa) [inline src/common/i18n/he.json:41]: spelling

File: src/common/i18n/he.json [hunk 4]
```diff
@@ -32,6 +33,16 @@
...
+    "export-table.table-column-header.link.text": "קישןר",
```
Comment (asafMasa) [inline src/common/i18n/he.json:42]: spelling

File: src/exporter/components/export-table/cell-renderer/link.cell-renderer.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+}
+a.buttonLink:hover{
+  border-color: rgba(255,255,255,1);
+}
+@media all and (max-width:30em){
+  a.buttonLink{
+  display:block;
+  margin:0.2em auto;
+  }
+}
```
Comment (asafMasa) [inline src/exporter/components/export-table/cell-renderer/link.cell-renderer.css:22]: end of file

File: src/exporter/components/export-table/cell-renderer/progress.cell-renderer.css [hunk 6]
```diff
@@ -0,0 +1,6 @@
+.rendererLabel{
+    font-size: 12px;
+    position: absolute;
+    left: 50%;
+    top: -4px;
+}
```
Comment (asafMasa) [inline src/exporter/components/export-table/cell-renderer/progress.cell-renderer.css:6]: end of file

File: package.json [hunk 7]
```diff
@@ -66,12 +66,15 @@
     "@types/node": "^12.12.44",
     "@types/react": "^16.9.35",
     "@types/react-dom": "^16.9.8",
+    "ag-grid-community": "^24.0.0",
+    "ag-grid-react": "^24.0.0",
```
Comment (asafMasa) [inline package.json:70]: why use ag-grid directly and not from shared components?

File: src/exporter/components/export-table/export-table-dialog.css [hunk 8]
```diff
@@ -0,0 +1,3 @@
+#exportTable .mdc-dialog__surface{
+    max-width: 1020px;
+}
```
Comment (asafMasa) [inline src/exporter/components/export-table/export-table-dialog.css:3]: end of file

File: src/exporter/models/exporterStore.ts [hunk 9]
```diff
@@ -86,9 +75,47 @@ export const exporterStore = types
         }
       }
     );
+    const getGeoPackages: () => Promise<void> = flow(
```
Comment (asafMasa) [inline src/exporter/models/exporterStore.ts:78]: rename to getExportTasks

File: src/exporter/models/geoPackage.ts [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+  fileName: types.string,
+  sizeEst: types.number,
+  tilesEst: types.number,
+  status: types.string,
+  link: types.string,
+  date: types.Date,
+  progress: types.number,
+});
+
+export interface IGeoPackage extends Instance<typeof geoPackage> {}
```
Comment (asafMasa) [inline src/exporter/models/geoPackage.ts:13]: end of file

File: src/exporter/models/geoPackage.ts [hunk 11]
```diff
@@ -0,0 +1,13 @@
+import { types, Instance } from 'mobx-state-tree';
+
+export const geoPackage = types.model({
```
Comment (asafMasa) [inline src/exporter/models/geoPackage.ts:3]: rename data and file to exportRequestStatus or exportTaskStatus

**Review by asafMasa — CHANGES_REQUESTED** [review]
good job!
please add the extra fields to the initial request (number of tiles, etc.)

File: src/common/i18n/he.json (hunk 1 above)
Comment (alebinson) [reply src/common/i18n/he.json:39]: Fixed

File: src/common/i18n/he.json (hunk 2 above)
Comment (alebinson) [reply src/common/i18n/he.json:40]: Fixed

File: src/common/i18n/he.json (hunk 3 above)
Comment (alebinson) [reply src/common/i18n/he.json:41]: For my opinion "סטטוס" is good, you proposition?
Probably the better way is to give English version to Technical writer for Hebrew translation at the END

File: src/common/i18n/he.json (hunk 4 above)
Comment (alebinson) [reply src/common/i18n/he.json:42]: Fixed

File: src/exporter/components/export-table/cell-renderer/link.cell-renderer.css (hunk 5 above)
Comment (alebinson) [reply src/exporter/components/export-table/cell-renderer/link.cell-renderer.css:22]: Done

File: src/exporter/components/export-table/cell-renderer/progress.cell-renderer.css (hunk 6 above)
Comment (alebinson) [reply src/exporter/components/export-table/cell-renderer/progress.cell-renderer.css:6]: Done

File: src/exporter/models/exporterStore.ts (hunk 9 above)
Comment (alebinson) [reply src/exporter/models/exporterStore.ts:78]: Done, but in other brunch unit tests will be effected (we need to minimize time to PR review)

File: src/exporter/models/geoPackage.ts (hunk 10 above)
Comment (alebinson) [reply src/exporter/models/geoPackage.ts:13]: Done

File: src/exporter/models/geoPackage.ts (hunk 11 above)
Comment (alebinson) [reply src/exporter/models/geoPackage.ts:3]: Renaming will be done in unit test PR, currently too much merging involved, sorry

File: src/exporter/models/exporterStore.ts [hunk 12]
```diff
@@ -86,9 +75,47 @@ export const exporterStore = types
...
+      function* getGeoPackages(): Generator<
+        Promise<GeoPackageResponse>,
+        void,
+        GeoPackageResponse
+      > {
+        try {
+          console.log('Fetch geoPackages--->');
```
Comment (asafMasa) [inline src/exporter/models/exporterStore.ts:85]: in master get from actual url, not a mock

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good job, please get the data from real url (config) instaed of mock

**Review by asafMasa — APPROVED** [review]

**Pull request merged**